configs/_base_/datasets/semantickitti.py

```diff
@@ -95,7 +94,10 @@
         file_client_args=file_client_args),
     dict(
         type='LoadAnnotations3D',
+        with_bbox_3d=False,
+        with_label_3d=False,
         with_seg_3d=True,
+        seg_3d_dtype='np.int32',
```

Comment:
Why modify the default type of data?

Reply:
semantickitti data.bin is saved from np.int32.